### DISCUSSION TITLE
'sea-orm-cli generate entity' failling with Planetscale DBs

### DISCUSSION THREAD

**Discussion by adragione** [SeaQL/sea-orm#1758] [General]
## Description

When attempting to update the local schema of my project based on latest DB changes that I had pushed to a MySQL DB hosted on Planetscale, I am suddenly receiving the error message mentioned below when running the 'sea-orm-cli generate entity' command. It seems that a panic is suddenly occurring when attempting to do this regular activity.

I have attempted with multiple databases/branches, as well as with a brand new DB stood up from the ground up. I was just able to successfully use the CLI to keep my schema in sync with the DB about a day or two back. Is it possible that some integration point with Planetscale is broken?

## Steps to Reproduce

1. Download latest version of sea-orm-cli (cargo install sea-orm-cli)
2. Gather connection string details provided by Planetscale for your MySQL database
3. Run the following command with your connection string details to generate the local schema files in your codebase:
    **sea-orm-cli generate entity -u mysql://<user_id>:password@<db_host>/<db_name> -o src/entities**

### Expected Behavior

I would expect to have my relevant schema files generated in the defined output location, 'src/entities'

### Actual Behavior

I'm getting the following error message when attempting to do this:

_thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ColumnDecode { index: "0", source: "mismatched types; Rust type `alloc::string::String` (as SQL type `VARCHAR`) is not compatible with SQL type `VARCHAR`" }', /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sqlx-core-0.6.3/src/row.rs:73:37
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace_

### Reproduces How Often

This is consistently reproducible (multiple DBs, environments, workstations, users)

### Workarounds

No real workarounds other than manually updating my schema files (which I'd like to avoid)

## Versions

sea-orm-cli v0.11.3

**Comment by tyt2y3** [comment]
I am sorry, but we are unable to devot resources to support proprietary, may-be-non-standard cloud database vendors. However, if you can do more digging and discovered the root cause, we can discuss the possible workarounds or bug fix.
